packages/pakistan/README.md

```diff
@@ -0,0 +1,181 @@
+# 🇵🇰 Paksitan
+
+Contains all the provinces of Pakistan:
+* Azad Jammu Kashmir
+* FATA
+* Gilgit Baltistan
+* Balochistan
+* Khyber Pakhtunkwah
+* Punjab
+* Sindh
+
+Contains all the cities of Pakistan:
+
+* Abbotabad
+* Astore
+* Attock
+* Awaran
+* Badin
+* Bagh
+* Bahawalnagar
+* Bahawalpur
+* Bajur
+* Bannu
+* Barkhan
+* Battagram
+* Bhakkar
+* Bhimber
+* Bolan
+* Buner
+* Chaghai
+* Chakwal
+* Charsadda
+* Chiniot
+* Chitral
+* Dadu
+* Dera Bugti
+* Dera Ghazi Khan
+* Dera Ismail Khan
+* Diamer
+* FR Bannu
+* FR D.I.Khan
+* FR Kohat
+* FR Lakki
+* FR Peshawar
+* FR Tank
+* Faisalabad
+* Ghanhce
+* Ghizer
+* Ghotki
+* Gilgit
+* Gujranwala
+* Gujrat
+* Gwadar
+* Hafizabad
+* Hangu
+* Haranai
+* Haripur
+* Hattian
+* Haveli
+* Hunza Nagar
+* Hyderabad
+* Islamabad
+* Jacobabad
+* Jafarabad
+* Jamshoro
+* Jhal Magsi
+* Jhang
+* Jhelum
+* Kalat
+* Karachi
+* Karak
+* Kashmore
+* Kasur
+* Kech
+* Khairpur
+* Khanewal
+* Kharan
+* Khushab
+* Khuzdar
+* Khyber
+* Killa Abdullah
+* Killa Saifullah
+* Kohat
+* Kohistan
+* Kohlu
+* Kotli
+* Kurram
+* Lahore
+* Lakki Marwat
+* Larkana
+* Lasbela
+* Layyah
+* Lodhran
+* Loralai
+* Lower Dir
+* Malakand
+* Mandi Bahauddin
+* Mansehra
+* Mardan
+* Mastung
+* Matiari
+* Mianwali
+* Mirpur
+* Mirpur Khas
+* Mohmand
+* Multan
+* Musakhel
+* Muzaffarabad
+* Muzaffargarh
+* Nankana Sahab
+* Narowal
+* Naseerabad
+* Naushahro Firoze
+* Nawabshah
+* Neelum
+* North Waziristan
+* Nowshera
+* Nushki
+* Okara
+* Orakzai
+* Pakpattan
+* Panjgur
+* Peshawar
+* Pishin
+* Poonch
+* Qambar Shahdatkot
+* Quetta
+* Rahimyar Khan
+* Rajanpur
+* Rawalpindi
+* Sahiwal
+* Sanghar
+* Sargodha
+* Shaikhupura
+* Shangla
+* Sheerani
+* Shikarpur
+* Sialkot
+* Sibi
+* Skardu
+* South Waziristan
+* Sudhnati
+* Sukkur
+* Swabi
+* Swat
+* Tando Muhammad Khan
+* Tando_Allahyar
+* Tank
+* Tharparkar
+* Thatta
+* Toba Tek Singh
+* Umerkot
+* Upper Dir
+* Vehari
+* Washuk
+* Zhob
+* Ziarat
```

Comment:
Could you use English and consistent names please? (in the SVG file too)
* Abbotabad -> Abbottabad
* FR Bannu -> Bannu
* Tando_Allahyar - Tando Allahyar
...